source/reporting/index.html.md.erb

```diff
@@ -89,16 +89,36 @@ have entered when making a payment:
     },
 ```
 
+### Checking when your PSP took a payment
+
+If your PSP is Worldpay, you can see when Worldpay took a payment from your user's bank account.
```

Comment:
Card, not bank account

Reply:
Hi Alex - we think it's best to stick with "bank account" here, as using "card" might leave some ambiguity about whether the user has actually had the payment taken from them. Hope that's ok.

Reply:
Hmm. For a credit card, the money won’t have been taken from the user’s bank account at that point (and might never be at all depending on how the user pays their credit card bill).

I could live with just “your user’s account”.

Reply:
Ah good point - I've changed to "your user's account" as suggested in https://github.com/alphagov/pay-tech-docs/pull/302/commits/e1f09a4321a173975596da6ade8547c5cf51c8e3. Please approve if everything looks ok now!